src/Orchard.Environment.Shell.Abstractions/Builders/ShellContext.cs

```diff
@@ -27,7 +27,7 @@ public class ShellContext : IDisposable
         /// </summary>
         public IServiceScope CreateServiceScope()
         {
-            return ServiceProvider.GetRequiredService<IServiceScopeFactory>().CreateScope();
+            return ServiceProvider.CreateScope();
         }
```

Comment:
This seems to be a new extension method from 1.1 that does exactly what I wrote. So I assume you can remove this method altogether.